src/components/Wizard/NewEditImportPane.tsx

```diff
@@ -108,11 +120,69 @@ function NewForm({ onNewMessage }: NewFormProps) {
   );
 }
 
-function EditForm() {
+function EditForm({ onEditMessage, onDeleteMessage, messages }: EditFormProps) {
+  const formContext = useForm<EditFormData>({ mode: "onChange" });
+  const {
+    register,
+    handleSubmit,
+    formState: { isDirty, isValid },
+  } = formContext;
+
+  const onSubmit: SubmitHandler<EditFormData> = (data: EditFormData) => {
+    onEditMessage(data.id);
+  };
+
+  const onDelete = (id: string) => {
+    onDeleteMessage(id);
+  };
   return (
     <>
       <Card.Title>Edit an Existing Message</Card.Title>
-      <Card.Text>There are no messages saved.</Card.Text>
+
+      <Form onSubmit={handleSubmit(onSubmit)} className="edit-form">
+        <FormProvider {...formContext}>
+          <Form.Group controlId="message-select">
+            <FormRow label="Select Message">
+              {Object.keys(messages).length ? (
+                Object.entries(messages).map(([id, message]) => {
+                  return (
+                    <div className="edit-entry" key={id}>
+                      <RegisteredFormCheck
+                        name="id"
+                        register={register}
+                        registerOptions={{ required: true }}
+                        type="radio"
+                        key={id}
+                        label={id}
+                        value={id}
+                        id={`message-select-${id}`}
+                      />
+                      <Button
+                        key={`delete-${id}`}
+                        type="button"
+                        bsPrefix="btn-del"
```

Comment:
this should be className. bsPrefix is used to generate classnames (e.g., `btn-del-primary`, `btn-del-outline`, etc.)